app/retail/views.py

```diff
@@ -1574,3 +1575,39 @@ def clean_url(url):
     }
 
     return TemplateResponse(request, 'admin_index.html', context)
+
+
+@cached_as(JSONStore.objects.filter(view='posts', key='posts'), timeout=1200)
+@cached_view
+def fetchPost(qt='2'):
+    jsonstores = JSONStore.objects.filter(view='posts', key='posts')
+    if jsonstores.exists():
+        return jsonstores.first().data
```

Comment:
this is needed here?

Reply:
oops nope... i put it there for reference... taking it out now